data/json/monsterdrops/zombie_prisoner.json

```diff
@@ -0,0 +1,15 @@
+[
+  {
+    "type": "item_group",
+    "subtype": "collection",
+    "id": "mon_zombie_prisoner_death_drops",
+    "entries": [
+      { "item": "jumpsuit", "damage": [ 1, 4 ] },
```

Comment:
Consider changing this to `striped_shirt` and `striped_pants` which characters with `Convict` profession are wearing.

Reply:
But do prisoners nowadays actually wear striped clothes? Don't they wear orange or red jumpsuits? I thought mostly they changed those clotes, especially in the US.

Reply:
On your consideration.

Reply:
For reference: https://github.com/CleverRaven/Cataclysm-DDA/pull/20306#pullrequestreview-22595499

Reply:
I can change the whole thing if that is the common consesus. I just never seen anybody wearing black and white striped prisoner clothes in any American movie made in the last 60 years.